README.md

```diff
@@ -111,7 +102,7 @@ Or if you want to run every 20s while filtering out any "arriving" records
 Use Task Scheduler.
 
 #### cronic.py
-If the above sounds complicated, here's a simple looping script that calls `ttc_api_scraper.py` every minute during the TTC's operating hours. Just start it in your command line with  
```

Comment:
The above red sections are to be deleted:
1,2,6,8,10,12,15,17-21,23,25,27,29-32,35-48,50,52,54-55?,58,60,64-68,
78-79,82-84,90-94,114
? review & confirm.

Reply:
Further to previous comments, 54-55 is also redundant.
Deleting the red version by cut & delete,  keys not working...